### PR TITLE
Update rules README

### PR DESCRIPTION
**Pull request by nlopezgi**
Was very much out of date

### CODE REVIEW

File: rules/README.md [hunk 1]
```diff
@@ -1,22 +1,14 @@
 # Usage
 
-This directory contains a skylark rule to generate the toolchain configs. See README.md under
-the root directory for the definition of toolchain configs.
+This directory contains a Starlark rules to generate toolchain configs that can be used
```
Comment (smukherj1) [inline rules/README.md]: `a Starlark rules` -> `Starlark rules`

File: rules/README.md [hunk 2]
```diff
@@ -1,22 +1,14 @@
...
+with remote execution. See README.md under the root directory for the definition of toolchain configs.
 
-For a given version of Bazel, the `docker_toolchain_autoconfig` rule will:
-* build the execution container image,
-* install Bazel,
-* build the container, and
-* extract the toolchain config.
+User facing rules in this repo are:
+* `rbe_autoconfig` rule (see rbe_repo.bzl)  is a repo rule that attempts to find toolchain configs that will work for your environment. If no toolchain configs are found, it pulls a container and generates them on demand. This rule also (optionally) can create the toolchain configs within your source tree.
```
Comment (alex1545) [inline rules/README.md]: Remove extra space before "is a repo..."

File: rules/README.md (hunk 1 above)
Comment (nlopezgi) [reply rules/README.md]: fixed

**Pull request merged**